src/Tests/Microsoft.NET.Build.Tests/GivenThatWeWantToBuildAWindowsDesktopProject.cs

```diff
@@ -341,6 +341,37 @@ public void SupportedOSPlatformVersionCanBeSetInDirectoryBuildTargets()
             GetPropertyValue(testAsset, "TargetPlatformMoniker").Should().Be("Windows,Version=10.0.19041.0");
         }
 
+        [WindowsOnlyTheory]
+        [InlineData("net5.0", true)]
+        [InlineData("net5.0-windows10.0.19041.0", true)]
+        [InlineData("netcoreapp3.1", false)]
+        [InlineData("net472", false)]
+        public void WindowsWorkloadIsInstalledForNet5AndUp(string targetFramework, bool windowsWorkloadInstalled)
```

Comment:
`windowsWorkloadInstalled` seems like the wrong parameter name here, as the workload is "installed" in all cases.  Would something like `supportsWindowsTargetPlatformIdentifier` work better?